addons/lunch/views/report_lunchorder.xml

```diff
@@ -26,31 +26,33 @@
                             </tr>
                         </thead>
                         <tbody>
-                            <t t-foreach="get_users(docs)" t-as="o">
+                            <t t-foreach="docs.read_group([('id', 'in', docs.ids)],['user_id'],['user_id'])" t-as="o">
+                                <t t-set="user" t-value="user.browse(o['user_id'][0])"/>
+                                <t t-set="lines" t-value="docs.search([('user_id', '=', user.id)])"/>
                                 <tr>
                                     <td colspan="2">
-                                        <strong t-field="o.name"/>
+                                        <strong t-field="user.name"/>
                                     </td>
                                     <td class="text-right" colspan="2">
                                         <strong>
-                                            <span t-esc="get_total(o, docs)"/>
-                                            <span t-field="o.company_id.currency_id.symbol"/>
+                                            <span t-esc="sum([line.price for line in lines])"/>
```

Comment:
Actually here the list marks ([ and ]) are not necessary, as the for line in lines will return an interable.